rasa/core/validator.py

```diff
@@ -1,13 +1,16 @@
 import logging
 from collections import defaultdict
-from typing import List, Set, Text
-
-from rasa.constants import DOCS_URL_DOMAINS, DOCS_URL_ACTIONS
-from rasa.core.constants import UTTER_PREFIX
+from typing import Set, Text, Optional
```

Comment:
I think this module shouldn't be in `rasa.core` since it's also validating NLU. However, best create an extra issue for this.

Reply:
What module do you mean?

Reply:
The `validator` module should be one level higher (e.g. `rasa.validator`)